packages/eslint-config-javascript/index.js

```diff
@@ -20,6 +20,13 @@ module.exports = {
         */
         'arrow-parens': ['error', 'as-needed'],
 
+        /*
+            Use the "one true brace style" in which in which the opening brace of a block is placed 
+            on the same line as its corresponding statement or declaration (this matches Airbnb).
+            Also require the body within the braces to be on a new line.
+        */
+        'brace-style': ['error', '1tbs', { allowSingleLine: false }],
```

Comment:
Use this rule that says not to use it :P https://typescript-eslint.io/rules/brace-style/#how-to-use